mvc/src/main/java/nextstep/mvc/view/JsonView.java

```diff
@@ -1,8 +1,33 @@
 package nextstep.mvc.view;
 
-public class JsonView extends AbstractView {
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import nextstep.web.support.MediaType;
 
-    public JsonView(String viewName) {
-        super(viewName);
+import java.io.PrintWriter;
+import java.util.Map;
+
+
+public class JsonView implements View {
+
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @Override
+    public void render(
+            Map<String, ?> model,
+            HttpServletRequest request,
+            HttpServletResponse response
+    ) throws Exception {
+        response.setContentType(MediaType.APPLICATION_JSON_UTF8_VALUE);
+        final PrintWriter writer = response.getWriter();
+        if (model.size() == 1) {
+            final Object value = model.values().toArray()[0];
+            writer.print(value);
+            return;
+        }
+        final String jsonValues = OBJECT_MAPPER.writeValueAsString(model);
+        writer.print(jsonValues);
```

Comment:
```suggestion
        Object value = model;
        if (model.size() == 1) {
            value = model.values().toArray()[0];
        }
        final String jsonValues = OBJECT_MAPPER.writerWithDefaultPrettyPrinter().writeValueAsString(value);
        writer.print(jsonValues);
```
우선 제가 생각하기로 요구사항이 model size가 1개든 그 이상이든 전부 json으로 변환시켜서 반환해야하는 걸로 알고있어요!
기존에는 value 자체를 writer에 담아서 print 시키셨길래 수정해주시면 좋을거 같아용
그리고 writer.print가 중복되어서 사용되는데 요렇게 바꿔봐도 괜찮을거 같네요ㅎㅎ 더 좋은 방법이 있으면 다르게 수정해도 괜찮고요!!
마지막으로 writerWithDefaultPrettyPrinter()를 사용하고 writeValue를 하면 json이 이쁘게 찍힐겁니당

Reply:
데이터가 하나면 그 자체로 반환한다고 생각했는데 그렇게 되니 객체는 객체 주소만 던져주더라고요ㅋㅋㅋ 그래서 오즈의 조언대로 변경했습니다ㅋㅋㅋㅋ